solutions/cses-1073.mdx

```diff
@@ -2,22 +2,37 @@
 id: cses-1073
 source: CSES
 title: Towers
-author: Benjamin Qi, Danh Ta Chi Thanh
+author: Nathan Gong, Benjamin Qi, Danh Ta Chi Thanh
 ---
 
-**Time Complexity:** $\mathcal{O}(N\log N)$
-
 Greedy approach: always add the next cube on top of the tower with the smallest
-possible cube (or create a new tower if this isn't possible).
+possible cube on top (or create a new tower if this isn't possible).
 
 Equivalent to longest non-decreasing subsequence!
 
+<Warning>
```

Comment:
i don't know if this should be a warning or not. because warnings can be done for like java not passing a test case because its too slow. but this can be part of the explanation

Reply:
Yeah true